charts/jenkins/values.yaml

```diff
@@ -16,15 +16,20 @@ clusterZone: "cluster.local"
 renderHelmLabels: true
 
 controller:
-  httpsKeyStore:
-    jenkinsHttpsJksSecretName: ''
-    enable: false
+  https:
```

Comment:
We should move this further down in `values.yaml`. Things like configuring the image are more important and should be before this.

Reply:
I'd propose we do this in a separate PR. I've started reordering values [here](https://github.com/wmcdona89/helm-charts/blob/reorder-values/charts/jenkins/values.yaml).